upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -107,20 +108,128 @@ backup of Workload Manager configuration data and files is created. Once complet
    /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prepare-assets.sh --csm-version csm-${CSM_RELEASE} --tarball-file "${CSM_TAR_PATH}"
    ```
 
-## Stage 0.2 - Update SLS
+## Stage 0.2 - Plan and coordinate network upgrade
 
 ### Abstract (Stage 0.2)
 
-CSM 1.2 introduces the bifurcated CAN (BICAN) as well as network configuration controlled by data in SLS. An offline upgrade of SLS data is performed. For more details on the
+Prior to CSM 1.2, the single Customer Access Network (CAN) carried both the admin network traffic and the user network
+traffic. CSM 1.2 introduces bifurcated CAN (BICAN), which is designed to separate admin network traffic and user network traffic.
+With BICAN, the pre-1.2 CAN network is split into two separate networks:
+
+1. Customer Management Network (CMN)
+
+   This network allows only system administrative access from the customer site. The pre-1.2 CAN is renamed to CMN. By
+   the end of the CSM 1.2 upgrade, all non-administrative access, such as from UANs, will be removed from CMN.
+
+   During the CSM 1.2 upgrade, UANs will retain their pre-1.2 CAN IPs in order to minimize disruption to UANs. However,
+   toward the end of the CSM 1.2 upgrade, UANs will stop registering themselves on CMN and will receive new IPs on the
+   CAN/CHN network below. This process is described in more detail in [UAN Migration](#uan-migration).
+
+1. Customer Access Network (CAN) / Customer High-speed Network (CHN)
+
+   For user traffic only (e.g. users running and monitoring jobs), CSM 1.2 allows you to pick one of two networks:
+
+      - Customer Access Network (CAN): this is a new network (VLAN6 in switches) that runs over the management network. This
+         network must not be confused with pre-1.2 CAN, which was a monolithic network that allowed both user and administrative
+         traffic, was configured as VLAN7 in switches, and is now renamed to CMN. The new CAN allows only user traffic.
+
+      - Customer High-speed Network (CHN): this is a new network (VLAN5 in switches) that runs over the high-speed fabric.
```

Comment:
Probably should explicitly say CHN is in preview for CSM 1.2.  We have a couple customers that will jump on the preview.  Most customers should be use the new CAN as the recommended upgrade.

Reply:
Thanks for that pointer. Will make that change.

Reply:
Added `[Recommended]` and `[CSM 1.2 Tech Preview]` next to CAN and CHN respectively. Also, reiterated afterwards with `Please note that CHN is a tech preview in CSM 1.2, and the new CAN is the recommended upgrade.`